### PR TITLE
Iteration0 testing

### PR DESCRIPTION
**Pull request by JoelSmith123**
Made snapshot tests for Header, SignUp, App, and MovieCard.
Wrote tests for IS_LOADING, HAS_ERRORED, and FETCH_DATA_SUCCESS action creators.

### CODE REVIEW

File: src/actions/index.test.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      type: 'IS_LOADING',
+      isLoading
+    }
+
+    const result = actions.isLoading(isLoading)
+
+    expect(result).toEqual(expected)
+  })
+
+  it('should return a type of HAS_ERRORED, with an id', () => {
```
Comment (kayleenovak) [inline src/actions/index.test.js]: Your implementation of this test looks great! The it block is a little misleading with stating 'with an id' when it is returning hasErrored.

File: src/actions/index.test.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      type: 'HAS_ERRORED',
+      hasErrored
+    }
+
+    const result = actions.hasErrored(hasErrored)
+
+    expect(result).toEqual(expected)
+  })
+
+  it('should return a type of FETCH_DATA_SUCCESS, with a filter', () => {
```
Comment (kayleenovak) [inline src/actions/index.test.js]: Here the it block states you are returning a filter, when it is returning movies. The test looks great!

File: src/actions/index.test.js (hunk 1 above)
Comment (JoelSmith123) [reply src/actions/index.test.js]: whoops, thanks for pointing that out! Fixed!

File: src/actions/index.test.js (hunk 2 above)
Comment (JoelSmith123) [reply src/actions/index.test.js]: good catch, thanks! Fixed!

File: src/components/App/App.test.js [hunk 3]
```diff
@@ -1,9 +1,30 @@
...
+        <Header />
+        <Route exact path="/" render={() => <MovieContainer movies={movies}/>}/>
+      </div>
+    )
+
+    expect(wrapper).toMatchSnapshot();
+  });
+})
+
+
```
Comment (Dhanciles) [inline src/components/App/App.test.js]: Great job on this Joel! For these test we don't actually need to use all the ui elements. You can just assign `wrapper` to `shallow(<App />)'  and that mocks the render of the component and then all we have to do is check that wrapper then matches the snapshot.

Comment (JoelSmith123) [reply src/components/App/App.test.js]: ah cool, I didn't realize that! Thanks Derek

**Comment by Dhanciles** [comment]
Looks good! Great job on this Joel!

**Comment by Dhanciles** [comment]
Great job on this Joel! I'll go ahead and merge this in!

**Pull request merged**